code/obj/item/gun/kinetic.dm

```diff
@@ -2716,6 +2767,12 @@ ABSTRACT_TYPE(/obj/item/survival_rifle_barrel)
 	default_magazine = /obj/item/ammo/bullets/lmg
 	ammobag_magazines = list(/obj/item/ammo/bullets/lmg)
 	ammobag_restock_cost = 3
+	recoil_stacking_enabled = TRUE
+	recoil_stacking_safe_stacks = 8
+	recoil_stacking_max_stacks = 16
+	recoil_stacking_amount = 1
+	recoil_max = 100
+
```

Comment:
feels like waaay too much recoil on this, considering it already has base spread.